inv/inventory.go

```diff
@@ -161,9 +178,16 @@ func (i *inventory) ReplaceAttributes(ctx context.Context, id model.DeviceID, up
 			}
 		}
 	}
-	if _, err := i.db.UpsertRemoveDeviceAttributes(ctx, id, upsertAttrs, removeAttrs); err != nil {
+
+	res, err := i.db.UpsertRemoveDeviceAttributes(ctx, id, upsertAttrs, removeAttrs, scope, etag)
```

Comment:
Pay attention: this store method also sets updated_ts, which we do not want to set when changing the tags.
The reason is that the updated_ts is also known as "latest check-in", and we use that as a cheap monitor solution for the devices. If you set the value when changing tags, we lose the information.

Reply:
Good point. Thanks!